kivymd/uix/refreshlayout/refreshlayout.py

```diff
@@ -237,7 +237,7 @@ class RefreshSpinner(ThemableBehavior, FloatLayout):
     Color of ellipse around the spinner in (r, g, b, a) or string format.
 
     :attr:`circle_color` is a :class:`~kivy.properties.ColorProperty`
-    and defaults to `[1, 1, 1, 1]`.
+    and defaults to `theme_cls.primary_dark`.
```

Comment:
By default to None

Reply:
yeah, but by default if `MDScrollViewRefreshLayout.circlre_color` equals `None`, then it's reassigned to `theme_cls.primary_dark`, so `RefreshSpinner.circle_color` is also `theme_cls.primary_dark`

Reply:
Must specify the actual value.

Reply:
ok, here you are

Reply:
```python
    spinner_color = ColorProperty()
    """
    Color of spinner in (r, g, b, a) or string format.
    :attr:`spinner_color` is a :class:`~kivy.properties.ColorProperty`
    and defaults to `[1, 1, 1, 1]`.
    """

    circle_color = ColorProperty(None)
    """
    Color of ellipse around the spinner in (r, g, b, a) or string format.
    :attr:`circle_color` is a :class:`~kivy.properties.ColorProperty`
    and defaults to `[1, 1, 1, 1]`.
    """
```

Reply:
you are probably viewing the first commit but should view the latest

Reply:
Please check your entire PR... You have a lot of typos there...

Reply:
uhhh.. typos like missed articles the or something else?

Reply:
https://user-images.githubusercontent.com/16930280/210007757-c0c7948e-ddab-47f1-be43-e9246af159a1.mov

Reply:
I have already told you about it. Default value of `ColorProperty()` without arguments is `[1, 1, 1, 1]`, so everything is fine with `spinner_color`. As for the `circle_color`, it's reassigned in line 199. Yeah, by default it's `None`, but cause it's for the docs, I strongly believe that it'll be better if `theme_cls.primary_darker` will remain

Reply:
Please correct the default values for the spinner_color and circle_color attributes